webviz_subsurface/plugins/_tornado_plotter_fmu/views/tornado_widget/_tornado_view.py

```diff
@@ -69,36 +69,36 @@ def __init__(
 
         viewcolumn = self.add_column(TornadoView.IDs.MAIN_COLUMN)
         first_row = viewcolumn.make_row()
-        first_row.add_view_element(TornadoViewElement(), TornadoView.IDs.TORNADO_WIDGET)
+        first_row.add_view_element(TornadoViewElement(), TornadoView.IDs.TORNADO_VIEW)
 
     def set_callbacks(self) -> None:
         @callback(
             Output(
-                self.view_element(TornadoView.IDs.TORNADO_WIDGET)
+                self.view_element(TornadoView.IDs.TORNADO_VIEW)
                 .component_unique_id(TornadoViewElement.IDs.BARS)
                 .to_string(),
                 "style",
             ),
             Output(
-                self.view_element(TornadoView.IDs.TORNADO_WIDGET)
+                self.view_element(TornadoView.IDs.TORNADO_VIEW)
                 .component_unique_id(TornadoViewElement.IDs.TABLE_WRAPPER)
                 .to_string(),
                 "style",
             ),
             Input(
-                self.view_element(TornadoView.IDs.TORNADO_WIDGET)
+                self.view_element(TornadoView.IDs.TORNADO_VIEW)
```

Comment:
Since you are still adding the settings group as a shared setting to the plugin, the callbacks must be moved to the first element that owns both children (the view and the shared settings group), i.e. the plugin. Moreover, the settings components (e.g. `PlotPicker.IDs.BARS_OR_TABLES`) are no children of your view elements but of the shared settings groups. Hence, all your IDs are incorrect. If you move the callbacks to the plugin class and replace the view elements with the respective shared settings groups, your plugin should work again.